client/packages/common/src/intl/locales/en/common.json

```diff
@@ -1448,8 +1448,10 @@
   "label.universal-code": "Universal code",
   "label.universal-name": "Universal name",
   "label.unusable-vvm-status": "Unusable VVM status",
+  "label.update-expected-delivery-date": "Update expected delivery date",
   "label.update-live-location": "Update live location",
   "label.update-transactions": "Update stock transactions",
+  "label.update-purchase-order-expected-delivery-date-for-all-lines": "This will update the expected delivery date for all the lines in this Purchase Order, do you want to continue?",
```

Comment:
Sorry John, this should only be updated for all _selected_ lines

Reply:
Thanks Roxy. Updated to only consider selected lines: https://github.com/msupply-foundation/open-msupply/pull/9244/commits/b6f80eb1859e94faaad7f4dc5dfbe9ca7b73dd45